### ISSUE TITLE
Improvement | Track time without task selection

### ISSUE THREAD

**Issue by karenever** [ever-co/ever-teams#3476]
Currently, it's not possible for a user to track time without selecting a task.

**Improvement**

Users should be able to start the timer without selecting a task
1. Remove the tooltip on the Play button when the user wants to start the timer without a task.
2. Task creation/selection dropdown should be empty by default, unless the user selects a task to track.
3. For users with enforced daily plans, the existing logic does not change.

**Comment by evereq** [comment]
@karenever for "Remove the tooltip on the Play button when the user wants to start the timer without a task.", it's important to notice that we don't need to remove that tooltip in case a daily plan is required. I.e. your 3) in the list talks about it, but developer can easy miss that. So in case when we enforcing daily plans, the tooltip should appear, but just ask user to create a plan to be able to track time. I also think that maybe instead of block button and show tooltip, it will be better to show that same popup you described in another task that explain user about need to create daily plan and have a button to do it, etc. What is your opinion on that? I.e. replace blocked timer button with tooltip with always unblocked button and popup with ACTIONS.

**Comment by karenever** [comment]
@evereq The daily plan logic already prevents the user from starting the timer by showing the create a plan popup, that is in place of the tooltip. my thinking is you can have daily plan and still start timer without tasks right now (in the future, we extend functionality to include "Requires Tasks" mode where some managers want both daily plan and task creation before starting timer). So in this case, it will still ask the user to create a plan first. Since we have that in the popup, that's why I said to remove the tooltip because the user will see the popup first by default.
